stylesheets/pages/products_list.css

```diff
@@ -0,0 +1,82 @@
+.as-aside {
+  width: 15rem;
+  padding-top: var(--sp-sm);
+  box-shadow: 0 0 2px var(--gray-500);
+}
+.as-aside > * {
+  padding: var(--sp-xsm) var(--sp-sm) var(--sp-sm) var(--sp-md);
+}
+.as-aside__button {
+  background-color: transparent;
+  border: none;
+  text-decoration: underline;
+}
+.as-filter__price,
+.as-filter__catag,
+.as-filter__rating {
+  box-shadow: 0 0 2px var(--gray-500);
```

Comment:
Suggestion: If you will use this box shadow property many times, then directly create a variable as `box-shadow-1` in your variable.css.

Reply:
Thanks for suggestion, Will surely create on going further